src/react-components/ui-root.js

```diff
@@ -623,6 +624,28 @@ class UIRoot extends Component {
       this.state.audioTrackClone.stop();
     }
 
+    constraints.audio.echoCancellation = true;
+    constraints.audio.noiseSuppression = true;
+    constraints.audio.autoGainControl = true;
```

Comment:
Should the preferences be used here, or are they only available for FxR?

Reply:
If I understand the `MediaTrackConstraints` correctly, the `getUserMedia` call below will only succeed if an audio track can be found with these constraints. Do we need a fallback in case _some_ audio track is available, but not one that is capable of `echoCancellation`, `noiseSuppression`, and `autoGainControl`?

Reply:
these flags are a no-op in the case that somehow they are not available, though I don't ever expect that to be the case. Strictly speaking, setting these 3 lines are not necessary as they default to true anyways, but I wanted to be explicit since it's not obvious that is what is happening.